tests/Unit/View/TemplatePathsTest.php

```diff
@@ -54,7 +54,8 @@ public function getSanitizePathTestValues()
             [__FILE__, strtr(__FILE__, '\\', '/')],
             [__DIR__, strtr(__DIR__, '\\', '/') . '/'],
             ['composer.json', strtr(getcwd(), '\\', '/') . '/composer.json'],
-            ['php://stdin', 'php://stdin']
+            ['php://stdin', 'php://stdin'],
```

Comment:
Can you add another valid URL which works out of the box, e.g. `file` or `data`? This way we ensure that indeed something besides `php://` works.